bin/http_server.re

```diff
@@ -46,8 +46,14 @@ let handle_request =
       | Ok(response) =>
         let response = E.response_to_yojson(response);
         await(Response.of_json(~status=`OK, response));
-      | Error(_err) =>
-        await(Response.make(~status=`Internal_server_error, ()))
+      | Error(err) =>
+        switch (err) {
+        | `Not_a_json => print_endline("Invalid json")
+        | `Not_a_valid_request(err) =>
+          print_endline("Invalid request:" ++ err)
+        | _ => print_endline("unhandled")
+        };
```

Comment:
That's gonna be helpful

Reply:
Cool. Shall I resolve this?

Reply:
GH is showing this comment as 'Requested Changes'. Not sure how to resolve this.